# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-05-23 13:37+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: fi\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

# Translated by DeepL.
#: admin_utils.py:12
msgid "Export selected objects as CSV file"
msgstr "Vie valitut kohteet CSV-tiedostona"

# Translated by DeepL.
#: admin_utils.py:110
msgid "CSV export"
msgstr "CSV-vienti"

# Translated by DeepL.
#: antivirus_utils.py:47
msgid "Your request was reported and if you send again infected files, you will be banned."
msgstr "Pyynnöstäsi ilmoitettiin, ja jos lähetät uudelleen saastuneita tiedostoja, sinut kielletään."

# Translated by DeepL.
#: antivirus_utils.py:48
msgid "Failed to run antivirus scanner."
msgstr "Virustentorjuntaohjelman suorittaminen epäonnistui."

# Translated by DeepL.
#: antivirus_utils.py:49
msgid "Cannot scan file because it does not exist."
msgstr "Tiedostoa ei voida skannata, koska sitä ei ole olemassa."

# Translated by DeepL.
#: antivirus_utils.py:50
msgid "Your file was rejected because it is infected."
msgstr "Tiedostosi hylättiin, koska se on saastunut."

# Translated by DeepL.
#: antivirus_utils.py:51
msgid "Cannot scan file because it is not a file."
msgstr "Tiedostoa ei voi skannata, koska se ei ole tiedosto."

# Translated by DeepL.
#: antivirus_utils.py:52
msgid "Cannot scan file because it is neither a file nor a directory."
msgstr "Tiedostoa ei voi skannata, koska se ei ole tiedosto eikä hakemisto."

# Translated by DeepL.
#: antivirus_utils.py:53
msgid "Failed to scan your file with the antivirus."
msgstr "Virustorjuntaohjelma ei skannannut tiedostoasi."

# Translated by DeepL.
#: auth_utils.py:40
msgid "Your account is disabled."
msgstr "Tilisi on poistettu käytöstä."

# Translated by DeepL.
#: auth_utils.py:42
msgid "Your username and password do not match."
msgstr "Käyttäjätunnus ja salasana eivät täsmää."

# Translated by DeepL.
#: auth_utils.py:44
msgid "Authentication required."
msgstr "Tunnistautuminen vaaditaan."

# Translated by DeepL.
#: auth_utils.py:47
msgid "Access authentication"
msgstr "Pääsyn todennus"

# Translated by DeepL.
#: auth_utils.py:83 auth_utils.py:88
msgid "The password must contain at least 3 types of characters (types are lower case letters, upper case letters, digits and special characters)."
msgstr "Salasanan on sisällettävä vähintään kolme merkkityyppiä (tyypit ovat pienet kirjaimet, isot kirjaimet, numerot ja erikoismerkit)."

# Translated by DeepL.
#: files_utils.py:48
msgid "B"
msgstr "B"

# Translated by DeepL.
#: files_utils.py:52
msgid "kB"
msgstr "kB"

# Translated by DeepL.
#: files_utils.py:56
msgid "MB"
msgstr "MB"

# Translated by DeepL.
#: files_utils.py:60
msgid "GB"
msgstr "GB"

# Translated by DeepL.
#: files_utils.py:63
msgid "TB"
msgstr "TB"

# Translated by DeepL.
#: forms_utils.py:119 settings_store/forms.py:48
msgid "Yes"
msgstr "Kyllä"

# Translated by DeepL.
#: forms_utils.py:121 settings_store/forms.py:50
msgid "No"
msgstr "Ei"

# Translated by DeepL.
#: json_utils.py:89
msgid "Invalid request method"
msgstr "Virheellinen pyyntömenetelmä"

# Translated by DeepL.
#: json_utils.py:94
msgid "Authentication required"
msgstr "Vaadittu tunnistautuminen"

# Translated by DeepL.
#: json_utils.py:125
msgid "Access denied"
msgstr "Pääsy evätty"

# Translated by DeepL.
#: json_utils.py:127
msgid "Page not found"
msgstr "Sivua ei löydy"

# Translated by DeepL.
#: json_utils.py:132
msgid "Infected file detected"
msgstr "Tartunnan saanut tiedosto havaittu"

# Translated by DeepL.
#: json_utils.py:137
msgid "Internal server error"
msgstr "Sisäinen palvelinvirhe"

# Translated by DeepL.
#: ldap_utils.py:74
msgid "Error when initializing connection with LDAP server."
msgstr "Virhe alustettaessa yhteyttä LDAP-palvelimeen."

# Translated by DeepL.
#: ldap_utils.py:74 ldap_utils.py:102 ldap_utils.py:129 ldap_utils.py:280
msgid "Error:"
msgstr "Virhe:"

# Translated by DeepL.
#: ldap_utils.py:102
msgid "Error when trying to authenticate on LDAP server."
msgstr "Virhe, kun yritetään todennusta LDAP-palvelimella."

# Translated by DeepL.
#: ldap_utils.py:128
msgid "Search on LDAP server failed."
msgstr "Haku LDAP-palvelimella epäonnistui."

# Translated by DeepL.
#: ldap_utils.py:131
msgid "Arguments:"
msgstr "Argumentit:"

# Translated by DeepL.
#: ldap_utils.py:165
msgid "User not found."
msgstr "Käyttäjää ei löydy."

# Translated by DeepL.
#: ldap_utils.py:177
msgid "Group not found."
msgstr "Ryhmää ei löydy."

# Translated by DeepL.
#: ldap_utils.py:280
msgid "Error when trying to get user information."
msgstr "Virhe, kun yritetään saada käyttäjätietoja."

# Translated by DeepL.
#: ldap_utils.py:289
msgid "Login failed (however the user exists in LDAP)."
msgstr "Sisäänkirjautuminen epäonnistui (käyttäjä on kuitenkin olemassa LDAP:ssa)."

# Translated by DeepL.
#: ldap_utils.py:291
msgid "Login succeeded."
msgstr "Sisäänkirjautuminen onnistui."

# Translated by DeepL.
#: ldap_utils.py:292
#, python-format
msgid "User \"%s\" data:"
msgstr "Käyttäjä \"%s\" data:"

# Translated by DeepL.
#: ldap_utils.py:297
msgid "User groups:"
msgstr "Käyttäjäryhmät:"

# Translated by DeepL.
#: ldap_utils.py:302
msgid "Error when trying to get groups."
msgstr "Virhe, kun yritetään saada ryhmiä."

# Translated by DeepL.
#: ldap_utils.py:313
msgid "No groups found."
msgstr "Ryhmiä ei löytynyt."

# Translated by DeepL.
#: magic_login/forms.py:11
msgid "Email address"
msgstr "Sähköpostiosoite"

# Translated by DeepL.
#: magic_login/forms.py:16
msgid "Invalid email address."
msgstr "Virheellinen sähköpostiosoite."

# Translated by DeepL.
#: magic_login/views.py:115
msgid "The submitted form is incorrect. Please correct all errors and send it again."
msgstr "Lähetetty lomake on virheellinen. Korjaa kaikki virheet ja lähetä se uudelleen."

# Translated by DeepL.
#: magic_login/views.py:128
msgid "The requested email address is not allowed."
msgstr "Pyydetty sähköpostiosoite ei ole sallittu."

# Translated by DeepL.
#: magic_login/views.py:137
#, python-format
msgid "Link to authenticate on %s"
msgstr "Linkki todennukseen %s"

# Translated by DeepL.
#: magic_login/views.py:148
msgid "Here is the link to log in on the site:"
msgstr "Tässä on linkki sivustolle kirjautumiseen:"

# Translated by DeepL.
#: magic_login/views.py:149
msgid "If you did not request this link, ignore this email."
msgstr "Jos et ole pyytänyt tätä linkkiä, jätä tämä sähköposti huomiotta."

# Translated by DeepL.
#: magic_login/views.py:162
msgid "Failed to send the email:"
msgstr "Sähköpostin lähettäminen epäonnistui:"

# Translated by DeepL.
#: magic_login/views.py:164
msgid "An email has been sent to you with the link to log in."
msgstr "Sinulle on lähetetty sähköpostiviesti, jossa on linkki kirjautumiseen."

# Translated by DeepL.
#: magic_login/views.py:169
msgid "Your session has expired, please get a new link to retry."
msgstr "Istuntosi on vanhentunut, hanki uusi linkki yrittääksesi uudelleen."

# Translated by DeepL.
#: magic_login/views.py:173
msgid "Your request is invalid."
msgstr "Pyyntösi on mitätön."

# Translated by DeepL.
#: settings_store/forms.py:81 settings_utils.py:64 settings_utils.py:125
msgid "No changes to save."
msgstr "Ei tallennettavia muutoksia."

# Translated by DeepL.
#: settings_store/forms.py:84
msgid "Your changes will be active in a few seconds."
msgstr "Muutokset ovat aktiivisia muutamassa sekunnissa."

# Translated by DeepL.
#: settings_utils.py:69 settings_utils.py:131
#, python-format
msgid "Invalid setting name: %s"
msgstr "Virheellinen asetusnimi: %s"

# Translated by DeepL.
#: settings_utils.py:79 settings_utils.py:141
msgid "Your changes were not saved but were applied to the running software."
msgstr "Muutoksesi eivät tallentuneet, mutta niitä sovellettiin käynnissä olevaan ohjelmistoon."

# Translated by DeepL.
#: settings_utils.py:113 settings_utils.py:170
msgid "Unable to write configuration file:"
msgstr "Konfiguraatiotiedoston kirjoittaminen ei onnistu:"

# Translated by DeepL.
#: settings_utils.py:115 settings_utils.py:172
msgid "Your changes have been applied. The service will be reloaded in a few seconds to make them effective."
msgstr "Muutoksesi on otettu käyttöön. Palvelu ladataan uudelleen muutaman sekunnin kuluttua, jotta muutokset tulevat voimaan."

# Translated by DeepL.
#: system_utils.py:93
msgid "Password required."
msgstr "Salasana vaaditaan."

# Translated by DeepL.
#: system_utils.py:102
#, python-format
msgid "Command exited with code %s."
msgstr "Komento päättyi koodilla %s."

# Translated by DeepL.
#: system_utils.py:134 system_utils.py:137 system_utils.py:143
#: system_utils.py:163
msgid "Unable to write file."
msgstr "Tiedoston kirjoittaminen ei onnistu."

# Translated by DeepL.
#: system_utils.py:134
msgid "Specified path is a directory."
msgstr "Määritetty polku on hakemisto."

# Translated by DeepL.
#: system_utils.py:137
msgid "Invalid file path."
msgstr "Virheellinen tiedostopolku."

# Translated by DeepL.
#: system_utils.py:149
msgid "You need to send the main password to edit this file."
msgstr "Sinun on lähetettävä pääsalasana, jotta voit muokata tätä tiedostoa."

# Translated by DeepL.
#: system_utils.py:157
#, python-format
msgid "Unable to create temporary file \"%s\"."
msgstr "Väliaikaista tiedostoa \"%s\" ei pystytä luomaan."

# Translated by DeepL.
#: system_utils.py:164
msgid "File updated."
msgstr "Tiedosto päivitetty."

# Translated by DeepL.
#: templates/forms_utils/password_toggle_input.html:5
msgid "Hide value"
msgstr "Piilota arvo"

# Translated by DeepL.
#: templates/forms_utils/password_toggle_input.html:5
#: templates/forms_utils/password_toggle_input.html:7
msgid "Edit value"
msgstr "Muokkaa arvoa"

# Translated by DeepL.
#~ msgid "Failed to backup settings:"
#~ msgstr "Asetusten varmuuskopiointi epäonnistui:"

# Translated by DeepL.
#~ msgid "Your changes will be active after the service restart."
#~ msgstr "Muutokset ovat aktiivisia palvelun uudelleenkäynnistyksen jälkeen."

# Translated by DeepL.
#~ msgid "The service will be restarted in 2 seconds."
#~ msgstr "Palvelu käynnistyy uudelleen 2 sekunnin kuluttua."
